Compliance/centurylink-audit-periods-cycles.md

```diff
@@ -0,0 +1,15 @@
+{{{
+  "title": "CenturyLink Audit Periods and Cycles",
+  "date": "03-24-2016",
+  "author": "Christian Brown",
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+Audit reports are conducted using data from the prior 12-month period, as an auditor has to have, at minimum, six months of data to be able to provide an opinion.
+
+For example, CenturyLink’s 2015 [SOC 2](/compliance/soc-2/) Report was conducted during the period of October 1, 2014 to September 30, 2015 and was available for distribution mid-November, taking into account audit draft time. CenturyLink followed up the SOC 2 report with a bridge letter, which was issued mid-January 2016 and covered October 1, 2015 through December 31, 2015. CenturyLink does not issue ad-hoc bridge letters at any other time. In order to provide this level of service, CenturyLink requires a mini-audit for each of the controls within the report before providing the findings to upper management for sign-off.
```

Comment:
Change URL to `https://www.ctl.io/compliance/soc-2/`
The `md` files in PublicKB are prcoessed differently than blog posts in PublicPlatform and DeveloperCenter